### DISCUSSION TITLE
Seren：真实立体的社交网络

### DISCUSSION THREAD

**Discussion by star8ks** [NonceGeek/Move-Camp#303] [Hackathon & Hackerhouse]
反共识观点：不是所有事情都是可以追求效率的，比如人与人之间的信任。

Seren 这个名字来自单词 Serendipity 的前五个字母。Serendipity 的含义是机缘，意外之喜，在这个语境里我们想用来形容人和人的相遇。每个人的出生，动人的友谊，甚至伟大的产品和合作，都来自于人的相遇。于万千人之中，没有早一步，也没有晚一步，你也恰好在这里，而且你还这么有趣，这样的相遇大概是人生最美好的经历之一，所以我们想把相遇用 SBT（Soulbound NFT） 的形式沉淀结晶下来。使用 NFT 的 metadata，构建真实立体的社交网络，信任体系，甚至和线下活动结合的邀请系统，票务系统等等。

这个产品的灵感来源于在 dDAO 中相遇的两个小伙伴，同在英国生活，最终在摩洛哥面基。她们都想把这次见面用有趣的，有仪式感的方式记录下来，但是苦于并没有适合的工具，所以就有了这个产品的想法。随着数字游牧，加密游牧的生活方式的流行，Seren 会成为连接起广大的社区，建立人的网络的有力工具。

即使 AI 将取代越来越多基础性工作，成为我们生活越来越重要的一部分，人与人之间真实的连接永远是无法被取代的，这种连接在自动的去中心化的区块链行业的世界里甚至变得更加宝贵。几年前 Vitalik 就是用四处拜访比特币社区的方式，团结凝结起了以太坊社区，改变了行业的生态。并没有太多产品能够为人类建立连接这种重要的体验提供支持，甚至通过设计和创新成为这种体验重要的一部分，所以我们认为 Seren 有潜力成为行业版图的重要部分。

Document every great encounter, your next "Seren" experience awaits.

**Comment by leeduckgo** [comment]
this proj could ref with:
> https://github.com/NonceGeek/Web3-dApp-Camp/discussions/287
